### PR TITLE
Automatically resolve DynamicShapeVariable to TensorVariable when calling torch.clamp

### PR DESCRIPTION
**Pull request by tugsbayasgalan**
Fixes: https://github.com/pytorch/torchdynamo/issues/911

### CODE REVIEW

File: test/test_export.py [hunk 1]
```diff
@@ -472,3 +472,15 @@ def func(a, b, c):
         dynamo_result = out_graph(*flat_input)
 
         self.assertTrue(torchdynamo.utils.same(real_result, dynamo_result))
+
+    def test_min_op(self):
```
Comment (voznesenskym) [inline test/test_export.py:476]: This is vague, as it uses export as a black box to enforce a guarantee. Instead, check out how our other test are written, in test_misc for example.

**Review by voznesenskym — CHANGES_REQUESTED** [review]

File: test/test_export.py [hunk 2]
```diff
@@ -472,3 +472,15 @@ def func(a, b, c):
...
+        torchdynamo.reset()
+        torchdynamo.config.dynamic_shapes = True
```
Comment (voznesenskym) [inline test/test_export.py:478]: Dynamic already has a test suite. Write the test for min without this.

**Comment by voznesenskym** [comment]
Nit: pr title should describe the fix

File: test/test_export.py (hunk 2 above)
Comment (tugsbayasgalan) [reply test/test_export.py:478]: I think Dynamic test suite just runs some test files under dynamic_shape = True mode. Isn't it better if I have actual test case for the min op (say inside test_misc.py). That way it is also tested under dynamic shapes.

Comment (voznesenskym) [reply test/test_export.py:478]: I believe we are saying the same thing.

Comment (tugsbayasgalan) [reply test/test_export.py]: addressed

**Comment by voznesenskym** [comment]
Pass lint, otherwise LGTM

**Review by voznesenskym — APPROVED** [review]

**Pull request merged**